mobile/package.json

```
@@ -115,7 +115,7 @@
     "lint:fix": "yarn build && yarn lint:fix:alone",
     "lint:fix:alone": "yarn lint:eslint --fix . && yarn lint:prettier --write .",
     "lint:prettier": "prettier  --config ../packages/tooling/.prettierrc --ignore-path ../packages/tooling/.prettierignore --loglevel warn",
-    "start:dev": "expo start",
+    "start:dev": "expo start --clear",
```

let's not clear each time, it will slowdown the boottime.

```suggestion
    "start:dev": "expo start",
    "start:dev:fresh": "expo start --clear",
```